exercises/01_intro/demo.R

```diff
@@ -0,0 +1,77 @@
+# Demo script for introduction session on 2021-09-16
+# We will transform Dotstat data to create figure 2.11 in OECD RCaG 2020
+# Disclaimer: This is an example for learning purposes, 
+# there will be some differences with the original figure
+
+## 0 Packages ----------------------------------------------------------------------------------------------------------
+
+# install.packages('tidyverse')
+library(tidyverse) # includes dplyr, tidyr, readr, ...
+# install.packages("writexl")
+library(writexl) # writing Excel files
+
+
+## 1 Data ----------------------------------------------------------------------------------------------------------
+## 2.1 Import ----------------------------------------------------------------------------------------------------------
+
+data <- read_csv("gva_per_worker.csv")
+
+# Alternative:
+# data <- readsdmx::read_sdmx("https://stats.oecd.org/restsdmx/sdmx.ashx/GetDataStructure/REGION_ECONOM")
+
+
+# Inspecting the data
+names(data) # overview of all variables in dataset
+head(data)
+# Inspecting variables
+unique(data$MEAS)
+
+
+## 2.2 Manipulation ----------------------------------------------------------------------------------------------------------
+# It is more convenient to work with all lower cap variable names
+data <- data %>% 
```

Comment:
Hi Jolien! I hope you don't mind, but I am going to change all mentions of `data <- data` in this script. It is generally considered bad practice to overwrite an existing object because it makes tracing back errors more difficult.

Reply:
Addressed in 31c4572

Reply:
Hi María Paula, Definitely, feel free to go ahead. I thought it would be easier for teaching but agree it's not good practice. Thanks, Jolien